### PR TITLE
Release 13.4.0

### PR DESCRIPTION
**Pull request by jsconan**
Please verify the following points :

- [x] the manifest (versions 13.4.0 and dependencies),
- [x] CSS and JavaScript bundles

### CODE REVIEW

File: common/oatbox/user/UserTimezoneService.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?php
+/**
```
Comment (mike-ionut-mihai-sandu-tao) [inline common/oatbox/user/UserTimezoneService.php:2]: please apply phpcs --standard=PSR12  on this

**Review by augustas — APPROVED** [review]
Approved

File: common/oatbox/user/UserTimezoneService.php [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+namespace oat\oatbox\user;
+
+
+use oat\oatbox\service\ConfigurableService;
+
+class UserTimezoneService extends ConfigurableService implements UserTimezoneServiceInterface
+{
+    public function getDefaultTimezone(): string
+    {
+        return TIME_ZONE;
```
Comment (mike-ionut-mihai-sandu-tao) [inline common/oatbox/user/UserTimezoneService.php:31]: what happens if the `TIME_ZONE` does not exist ? 
should we have something like 
```
defined(TIME_ZONE) ?? 'somedefault value '
```

**Review by mike-ionut-mihai-sandu-tao — CHANGES_REQUESTED** [review]

File: migrations/Version202009081435472348_generis.php [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+ * index.php '\oat\tao\scripts\tools\Migrations' -c execute -v 'oat\generis\migrations\Version202009081435472348_generis'
+ * rollback Migration
+ * php index.php '\oat\tao\scripts\tools\Migrations' -c rollback -v 'oat\generis\migrations\Version202009081435472348_generis'
+ */
+final class Version202009081435472348_generis extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Register UserTimezoneService';
```
Comment (mike-ionut-mihai-sandu-tao) [inline migrations/Version202009081435472348_generis.php:41]: maybe this should be handled as  a translated string

File: test/bootstrap.php [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA (under the project TAO-PRODUCT);
+ *
+ * @license    GPLv2
+ * @package    package_name
+ * @subpackage
+ */
+
+$extensionRoot = __DIR__ . '/../';
```
Comment (mike-ionut-mihai-sandu-tao) [inline test/bootstrap.php:24]: dirname(__DIR__)

File: test/bootstrap.php [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+// Use TAO dependency resolver first.
+require_once $extensionRoot . './../vendor/autoload.php';
```
Comment (mike-ionut-mihai-sandu-tao) [inline test/bootstrap.php:27]: DIRECTORY_SEPARATOR .. so it would work on windows also

File: manifest.php [hunk 6]
```diff
@@ -32,7 +32,7 @@
     'label' => 'Generis Core',
     'description' => 'Core extension, provide the low level framework and an API to manage ontologies',
     'license' => 'GPL-2.0',
-    'version' => '13.2.0',
+    'version' => '13.4.0',
```
Comment (gabrielfs7) [inline manifest.php:35]: Wondering why jumped 2 versions.

File: common/oatbox/user/UserTimezoneService.php (hunk 2 above)
Comment (gabrielfs7) [reply common/oatbox/user/UserTimezoneService.php:31]: I suggest insert this as an option to avoid global dependencies on our projects.

**Pull request merged**